src/inmanta/data.py

```diff
@@ -1659,15 +1658,27 @@ def get_increment(self, negative: bool=False):
         error -> increment
         Deployed and same hash -> not increment
         deployed and different hash -> increment
-         """
-        projection_a = {"resource_id": True, "status": True, "attribute_hash": True, "attributes": True}
-        projection = {"resource_id": True, "status": True, "attribute_hash": True}
+        """
+        projection_a = {
+            "resource_version_id": True,
+            "resource_id": True,
+            "status": True,
+            "attribute_hash": True,
+            "attributes": True
+        }
+        projection = {
```

Comment:
Only "resource_id" should be sufficient. Right?

Reply:
line 1732 also requires resource_version_id